Integrations/ThreatQ_v2/ThreatQ_v2.py

```diff
@@ -552,16 +561,28 @@ def add_malicious_data(generic_context, tq_score):
     }
 
 
-def set_indicator_entry_context(indicator_type, raw, dbot, generic):
-    if dbot.get('Score') == 3:
-        add_malicious_data(generic, raw.get('TQScore', -1))
+def set_indicator_entry_context(indicator_type, raws, dbots, generic):
```

Comment:
there is an implicit assumption that in every index in raw there is the corresponding value in dbots. I think this is confusing. better just pass an list of tuples.